test_scripts/Resumption/Handling_errors_from_HMI/commonResumptionErrorHandling.lua

```diff
@@ -32,21 +37,29 @@ m.cloneTable = utils.cloneTable
 m.wait = utils.wait
 m.tableToString = utils.tableToString
 m.connectMobile = actions.mobile.connect
+m.getModuleControlData = rc.predefined.getModuleControlData
+m.isTableEqual = utils.isTableEqual
+m.getActualModuleStateOnHMI = rc.state.getActualModuleStateOnHMI
+m.getActualModuleIVData = rc.state.getActualModuleIVData
 
 m.hashId = {}
 m.resumptionData = {
   [1] = {},
   [2] = {}
 }
 
+local moduleTypeValue = "CLIMATE"
```

Comment:
@GetmanetsIrina As I can see this is default module type variable, so I would propose to rename it
```suggestion
local defaultModuleType = "CLIMATE"
```